index.css

```diff
@@ -0,0 +1,4 @@
+body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Please consider adding at least one `CSS` class with styles for your `h1` header, as already pointed out by the previous reviewer this is part of the [Project Requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)